### PR TITLE
Change default instance to leetphoria

### PR DESCRIPTION
**Pull request by jeremyredhead**
Euphoria is dead, long live euphoria!

I didn't bother editing the references to api.euphoria.io (since leet doesn't have the API docs back up yet), 
nor the mention in the package setup.py (b/c I probably won't maintain this fork forever)

(yes, i *could* just commit this myself but i wanna make sure i'm not overlooking something)

### CODE REVIEW

**Comment by jeremyredhead** [comment]
@CylonicRaider could you review this? thanks

File: yaboli/room.py [hunk 1]
```diff
@@ -31,7 +31,7 @@ class Room:
 	reconnect a few times. Loss of connection is determined by self._conn.
 	"""
 	
-	ROOM_FORMAT = "wss://euphoria.io/room/{}/ws"
+	ROOM_FORMAT = "wss://euphoria.leet.nu/room/{}/ws"
```
Comment (CylonicRaider) [inline yaboli/room.py:34]: LGTM

**Review by CylonicRaider — APPROVED** [review]
P.S. That branch name is on fire.

P.P.S. If only I knew what that branch name means...

**Comment by jeremyredhead** [comment]
The branch is referencing a quote from *[It's a Wonderful Life](https://en.wikipedia.org/wiki/It%27s_a_Wonderful_Life)* in which the protagonist's daughter Zuzu asks him to paste the petals that have fallen off her rose back on it, but he actually just pockets them instead and lies (she's liek 5 and so doesn't know any better). Not sure what I was trying to say with the branch name, was mostly just feeling very punchy lol

**Pull request merged**